packages/configuration-development/package.json

```
@@ -13,6 +13,7 @@
     "@hint/hint-css-prefix-order": "^1.4.6",
     "@hint/hint-disown-opener": "^4.0.2",
     "@hint/hint-highest-available-document-mode": "^5.0.2",
+    "@hint/hint-leading-dot-classlist": "^1.0.0",
```

Go ahead and leave these at v0.0.1 since we haven't published to npm yet. Our release script will take care of updating these to 1.0.0 when we get ready to publish.
```suggestion
    "@hint/hint-leading-dot-classlist": "^0.0.1",
```